conf/defaults.ini

```diff
@@ -1110,6 +1110,10 @@ global_file = 1000
 # global limit of correlations
 global_correlations = -1
 
+# Limit of the number of alert rules per rule group.
+# This is not strictly enforced yet, but will be enforced over time.
+alerting_rule_group_rules = 100
```

Comment:
this is a little bit confusing, the setting is part of the quota settings but in fact is not enforced by the quota service. I wonder if it's place is in UA section. WDYT?

Reply:
Yeah...went back and forth on this one. I figured the average user might expect all the limits to be in one spot in the config, regardless of what enforces things internally.

The other side of this, is that this will probably never be enforced by quota service, even in the future, because of quota service architecture. It stores all the current quota values both in database and in memory... a new quota per rule group is a massive explosion in the number of objects the service has to manage.

I'll try moving it, let's see what we think :+1: